doc/changes/changes_4.4.3.md

```diff
@@ -5,6 +5,7 @@ Code name: Internal refactorings on "More Predicates"
 ## Refactoring
 
 * [#117](https://github.com/exasol/sql-statement-builder/issues/117): Fixed sonar findings
+* [#102](https://github.com/exasol/sql-statement-builder/issues/102): Refactored ColumnsDefinitionRenderer
```

Comment:
This won't be validated by Release Droid because it's not the right format. Normally Release Droid checks if the mentioned tickets are closed. We should either agree on a new format and use it across the team + change release droid a little bit or follow what we have right now

Reply:
Current format: `* #[ticket number]:`